chapters/001-getting-started/README.md

```diff
@@ -1,44 +1,57 @@
-TODO: modify all /bin/sh to /bin/bash
-
 # Getting Started
 
 ## Goals for this section:
-We want to build a simple network where two machines can ping each other. To keep this simple and remote pairing friendly, we want to use docker containers to simulate machines on this network. We will use some simple common networking tools to understand the shape of our network and how and when successful communication is occurring.
+
+We want to build a simple network where two machines can ping each other. To
+keep this simple and remote pairing friendly, we want to use docker containers
+to simulate machines on this network. We will use some simple common networking
+tools to understand the shape of our network and how and when successful
+communication is occurring.
+
+## Vocabulary
+
+- `container`: a running instance of a docker image. Each container we create in building out our internet will be one machine on that internet.
+- `machine`: any computer. Could be a server, a router, a client...
+- `host`: see `machine`.
```

Comment:
Why have two words for the same thing? Should we just be disciplined in our note-taking?  (I mean, we could add "box" and "node" and "system" for that matter 😉) 

I suppose I think of a "host" as distinct from a "gateway". In other words, "hosts do not route." Whereas a "machine" can be either one.

Reply:
Oooh! I like that distinction. Let me add that to the vocabulary and then make sure I'm consistent with it. 

I like having all the possible words in the vocabulary because people will come across them in their own explorations. I'd like them to have some concept of what they mean and context for how/when to use them. I'm going to keep this for now, but we can always remove later if y'all think it's just confusing.